mountpoint-s3/tests/fuse_tests/cache_test.rs

```diff
@@ -143,6 +149,62 @@ fn disk_cache_write_read(key_suffix: &str, key_size: usize, object_size: usize)
     );
 }
 
+#[tokio::test]
+async fn express_cache_verify_fail_non_express() {
+    let client = create_crt_client(CLIENT_PART_SIZE, CLIENT_PART_SIZE, Default::default());
+    let bucket_name = get_standard_bucket();
+    let cache_bucket_name = get_standard_bucket();
+    let cache = ExpressDataCache::new(client.clone(), Default::default(), &bucket_name, &cache_bucket_name);
+    let err = cache
+        .verify_cache_valid()
+        .await
+        .expect_err("cannot use standard bucket as shared cache");
+
+    if let ObjectClientError::ClientError(ResponseError(request_result)) = err {
+        let body = request_result.error_response_body.as_ref().expect("should have body");
+        let body = body.clone().into_string().unwrap();
+        assert!(body.contains("<Code>InvalidStorageClass</Code>"));
```

Comment:
how do we make S3 to return an error in this case?

Reply:
We're sending expected storage class is express onezone, which standard doesn't support, and thus fails with an invalid storage class error